robottelo/test.py

```diff
@@ -469,9 +501,47 @@ def _join_all_threads(self, thread_list):
         for thread in thread_list:
             thread.join()
 
-    def _write_raw_csv_file(self, raw_file_name, time_result_dict,
-                            current_num_threads, test_case_name):
-        """Write raw timing ak/del results to csv file"""
+    def _get_output_filename(self, file_name):
+        """Get type of test: ak/att/del/reg as output file name
+
+        Extract type of test cases, thus differentiate output
+        charts and csv files.
+
+        :param str file_name: File name is value of `self.raw_file_name`
+            or `self.stat_file_name`. For example:
+            file_name = 'perf-raw-activationKey.csv'
+        :return: file name before dot. For example: 'perf-raw-activationKey'
+        :rtype str
+
+        """
+        split_file_name = file_name.split('.')
+        return split_file_name[0]
+
+    def _write_raw_csv_file(
+            self,
+            raw_file_name,
+            time_result_dict,
+            current_num_threads,
+            test_case_name):
+        """Write raw timing ak/att/del/reg results to csv file
+
+        Common method shared by three test cases:
+            1. concurrent subscription by activation key
```

Comment:
Place lists at the same indentation level as surrounding prose. Separate lists from surrounding prose with a blank line.